projects/testing-library/tests/render.spec.ts

```diff
@@ -296,3 +297,49 @@ describe('DebugElement', () => {
     expect(view.debugElement.componentInstance).toBeInstanceOf(FixtureComponent);
   });
 });
+
+describe('initialRoute', () => {
+  @Component({
+    standalone: true,
+    selector: 'atl-fixture2',
+    template: `<button>Secondary Component</button>`,
+  })
+  class SecondaryFixtureComponent {}
+
+  @Component({
+    standalone: true,
+    selector: 'atl-router-fixture',
+    template: `<router-outlet></router-outlet>`,
+    imports: [RouterModule],
+  })
+  class RouterFixtureComponent {}
+
+  @Injectable()
+  class FixtureResolver implements Resolve<void> {
+    public isResolved = false;
+
+    public resolve() {
+      this.isResolved = true;
+    }
+  }
+
+  it('allows initially rendering a specific route to avoid triggering a resolver for the default route', async () => {
+    const expectedRoute = 'correct-route';
```

Comment:
Could you rename this to the following please.
I do think it makes it intention clearer.

```suggestion
    const initialRoute = 'initial-route';
```